testing/regress/ecl/soapcall.ecl

```diff
@@ -15,9 +15,6 @@
     limitations under the License.
 ############################################################################## */
 
-//nohthor
-//nothor
-
```

Comment:
Note I'd like this to run on all 3 at least initially, unless it causes problems.
We can remove thor and hthor again before merging if it's an issue.